### PR TITLE
mcomix3: manpage location, app icon

### PR DESCRIPTION
**Pull request by con-f-use**
###### Motivation for this change
Application icon is currently missing in `mcomix3`. Additionally, `man mcomix3` is failing, because the corresponding manpage is named `mcomix` instead of `mcomix3`. 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/graphics/mcomix3/default.nix [hunk 1]
```diff
@@ -56,13 +56,15 @@ python3.pkgs.buildPythonApplication rec {
   postInstall = ''
     rmdir $libdir/mcomix/mcomix
     cp man/* $out/share/man/man1/
+    mv $out/share/man/man1/mcomix.1 $out/share/man/man1/${pname}.1
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/mcomix3/default.nix:59]: Please use installManPage.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 105450` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mcomix3</li>
  </ul>
</details>

File: pkgs/applications/graphics/mcomix3/default.nix (hunk 1 above)
Comment (con-f-use) [reply pkgs/applications/graphics/mcomix3/default.nix:59]: But that wouldn't fix the name, and I'd need the `mv` with the hardcoded path anyway.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 105450` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mcomix3</li>
  </ul>
</details>

File: pkgs/applications/graphics/mcomix3/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/applications/graphics/mcomix3/default.nix:59]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/install-shell-files.sh

Comment (SuperSandro2000) [reply pkgs/applications/graphics/mcomix3/default.nix:59]: btw are there multiple man pages? if not cp and mv can be reduced to only mv.

Comment (con-f-use) [reply pkgs/applications/graphics/mcomix3/default.nix:59]: There are two. One for the thumbnailer and one for the viewer.

**Comment by con-f-use** [comment]
@SuperSandro2000 can we get this merged now?

File: pkgs/applications/graphics/mcomix3/default.nix [hunk 2]
```diff
@@ -55,20 +57,22 @@ python3.pkgs.buildPythonApplication rec {
 
   postInstall = ''
     rmdir $libdir/mcomix/mcomix
-    cp man/* $out/share/man/man1/
+    installManPage man/*
+    mv $out/share/man/man1/mcomix.1 $out/share/man/man1/${pname}.1
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/mcomix3/default.nix:61]:
```suggestion
    mv man/mcomix.1 man/${pname}.1
    installManPage man/*
```

I am not sure if this messes with compression or anything else.

**Pull request merged**